README.md

```diff
@@ -240,17 +242,24 @@ platforms that could be forked and stuffed.
 - [New questions](https://github.com/xxh/xxh/wiki#new-questions)
 
 ## Development
-In the [xxh development environment](https://github.com/xxh/xxh/tree/master/xde) there is full [dockerised](https://www.docker.com/) environment 
+
+### xxh Development Environment
+
+In the [xxh development environment](https://github.com/xxh/xxh/tree/master/xde) there is full [dockerised](https://www.docker.com/) environment
 for development, testing and contribution. The process of testing and development is orchestrated by `xde` tool and is as 
 easy as possible.
 
-**We have teams.** If you're in a team it does not mean you have an obligation to do something. The main goal of teams is to create groups 
+### Vagrant based Plugin Development
+
+To develop plugins, [Vagrant](https://www.vagrantup.com) supports starting [many configurations](https://app.vagrantup.com/boxes/search) of 
```

Comment:
Hi @drmikecrowe ! It looks like this sentence isn't complete.

Reply:
Doh, I thought I finished that section.  1s